multi.go

```diff
@@ -36,17 +37,19 @@ func (builder) Build(target resolver.Target, cc resolver.ClientConn, opts resolv
 		cc: cc,
 	}
 	var mr multiResolver
-	for _, t := range strings.Split(target.Endpoint, ",") {
+	for _, t := range strings.Split(target.Endpoint(), ",") {
 		parsedTarget := ParseTarget(t)
-		resolverBuilder := resolver.Get(parsedTarget.Scheme)
+		resolverBuilder := resolver.Get(parsedTarget.URL.Scheme)
 		if resolverBuilder == nil {
 			parsedTarget = resolver.Target{
-				Scheme:   resolver.GetDefaultScheme(),
-				Endpoint: t,
+				URL: url.URL{
```

Comment:
This is not exactly what gRPC itself does, would the difference be dangerous?

https://github.com/grpc/grpc-go/commit/aaff9e7ab90624f6be3da4555eeb27c68ba29e9d#diff-ce683c132234ad155738e60c1fabf51c62d442813d0d650662de4ff991cebfa3R1640

Reply:
You are right, it is best to defer to same approach.